sdk/tests/test_restrict_transactions_between_effective_dates.py

```diff
@@ -0,0 +1,81 @@
+import unittest
+import pytz
+import finbourne_access
+import json
+
+from datetime import datetime, timedelta
+from finbourne_access import api as sa
+from finbourne_access import models as access_models
+from finbourne_access.utilities import ApiClientFactory
+
+
+class FinbourneAccessTests(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        api_client = ApiClientFactory()
+        cls.policies_api = api_client.build(sa.PoliciesApi)
+
+    def test_restrict_transactions_between_effective_dates(self):
+
+        """
+        This policy allows you to restrict the creation, deletion or updating of
+         transactions between two effective dates (in this case 1 May 2022 to 1 June 2022).
+
+        We specify the effective date restriction in the 'for' attribute of the PolicyCreationRequest
+        model.
+        """
+
+        scope = "ibor"
+
+        policy_code = f"restrict-transactions-on-scope-{scope}-for-may-2022"
+
+        deactivate = datetime(9999, 12, 31, tzinfo=pytz.utc)

```

Comment:
this looks like its only used in one place below, can we inline to make it easier to read/follow

Reply:
Sure, done.